backend/de.metas.handlingunits.base/src/test/java/de/metas/handlingunits/impl/HUPickingSlotTest.java

```diff
@@ -53,6 +53,7 @@ public final void init()
 		AdempiereTestHelper.get().init();
 		
 		SpringContextHolder.registerJUnitBean(new PickingCandidateRepository());
+		SpringContextHolder.registerJUnitBean((PickingSlotConnectedComponent)slotId -> false);
```

Comment:
why do we need this?
what's the reason of returning false?

Reply:
to maintain the existing behaviour of the tests